Kudu.Services.Web/Content/Scripts/ProcessExplorer.js

```diff
@@ -943,23 +960,73 @@ function searchForHandle() {
     }
 }
 
-function handleProfilingEvents(e, processId) {
+function handleProfilingEvents(e, processId, iisProfilingTimeoutInSeconds) {
     if (!e || !e.target || typeof e.target.textContent === "undefined") {
         return;
     }
 
     var iisProfiling = false;
 
-    if (document.getElementById(processId + "-iisProfilingCheck") != null) {
-        iisProfiling = document.getElementById(processId + "-iisProfilingCheck").checked;
+    var iisProfilingCheckbox = document.getElementById(processId + "-iisProfilingCheck");
+
+    if (iisProfilingCheckbox != null) {
+        iisProfiling = iisProfilingCheckbox.checked;
     }
 
     if (e.target.textContent.indexOf("Stop") === 0) {
         stopProfiling(e, processId);
+        if (iisProfilingCheckbox != null) {
+            iisProfilingCheckbox.disabled = "";
+        }
     }
-    else if(e.target.textContent.indexOf("Starting") !== 0) {
-        e.target.textContent = "Starting Profiler...";
-        startProfiling(e, processId, iisProfiling);
+    else if (e.target.textContent.indexOf("Starting") !== 0) {
+
+        if (iisProfiling) {
+
+            var divConfirm = document.getElementById('dialog-confirm')
+            if (divConfirm == null) {
+                divConfirm = Utilities.createDiv('dialog-confirm');
+                divConfirm.title = "Collect IIS Events?";
+
+            }
+
+            divConfirm.innerHTML = "IIS profiling enables IIS and threadtime ETW events. The generated trace file can be analyzed using Perview which is available at <a style='outline: none' href='https://www.microsoft.com/en-us/download/details.aspx?id=28567'>https://www.microsoft.com/en-us/download/details.aspx?id=28567</a>. <br/><br/>The profiling session will automatically timeout after <b>" + iisProfilingTimeoutInSeconds.toString() + "</b> seconds if not stopped manually.<br/><br/>Enabling IIS Profiling is a relatively <b>expensive option</b> to turn on as it increases the CPU usage and disk I/O on your instance. Are you sure you want to continue ?"
+
+            $(divConfirm).dialog({
+                resizable: false,
+                height: 330,
```

Comment:
Nit: not a javascript expert but are we supposed to hard-code the sizes here? how does it show on different resolutions of hard coded? maybe check rest of Kudu, if they are same then ok.